docs/triggers/link-triggers.md

```diff
@@ -0,0 +1,120 @@
+## Link Triggers
+
+A Link Trigger is an interactive Trigger that activates when a Shortcut is clicked in the Slack client. When a Shortcut Trigger is created, its API response returns a `shortcut_url` which can be used in Slack to show a button. Clicking on this button will activate the associated Workflow. A Link Trigger
+includes the common [Trigger attributes](./trigger-basics.md#trigger-types) along with an optional shortcut parameter: 
+
+| Parameter name  | Required?     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| `shortcut`        | No            | Contains information about the [button text](#shortcut-object) of the shortcut          |
+
+### Shortcut Configuration
+
+A Link Trigger can contain an optional `shortcut` configuration object which specifies additional details about the Shortcut. Currently, the `shortcut` object is used to specify the button text of the Shortcut associated with the Trigger and has the following shape:
+
+```ts
+  shortcut?: {
+    button_text: string;
+  };
+```
+
+### Link Data Context Object
```

Comment:
Let's consistently name this section across the docs and link back to the `trigger-basics` file. I like "Context Data Availability"